app/presenters/hyrax/collapsable_section_presenter.rb

```diff
@@ -0,0 +1,50 @@
+module Hyrax
+  # Draws a collapsable list widget using the Bootstrap 3 / Collapse.js plugin
+  class CollapsableSectionPresenter
+    def initialize(view_context, text, id:, icon_class:, open:)
```

Comment:
One could provide a default, but it is not needed in our case. We are always passing something.